deltacat/utils/pyarrow.py

```diff
@@ -253,6 +254,41 @@ def s3_file_to_table(
         f"Reading {s3_url} to PyArrow. Content type: {content_type}. "
         f"Encoding: {content_encoding}"
     )
+
+    if content_type == ContentType.PARQUET.value:
+        from daft.table import Table
```

Comment:
This can be abstracted to a separate method.

Reply:
Factored out this code into a separate function in `util.daft`